### ISSUE TITLE
Can we propose translations to existing definitions? 

### ISSUE THREAD

**Issue by Tagada85** [tatianamac/selfdefined#70]
Hello!
I am learning a lot of things with self defined, even if the definitions are not in my native language, French. I was wondering if it was possible to propose translations for the existing definitions? In my case, I would be glad to help for the French ones. 

I assume you would need several people speaking French (in this case) to review each proposed translation. Is it something that could be interesting? 

Cheers! ❤️

**Comment by hibaymj** [comment]
Whoa! I was pretty sure I was off in left field when I put it in the API spec. I can't speak to the dictionary itself, but I did at that to the API design.

**Comment by tatianamac** [comment]
Yes! @Tagada85 ! I've had a couple of people propose offering translation for the dictionary—this is awesome to hear! 

Needs:
- **Find designated French editor/reviewer(s),** as you mentioned. As of right now, I fulfill this role for English definitions. I would want someone who could do this ongoing to keep the French translations consistent.
- **Determine how languages impacts current definition structure** (_does it become a part of the existing en markdown file, is it its own separate one?_)
- **Figure out how languages are accessed.** (_Is it a toggle for the whole site? [seems too premature for that] Does every definition need an English counterpart? Is there a need to view all terms in all languages anywhere?_)
- **How is this handled for the future API?** (some of this has been considered by @hibaymj's work 🙏🏽)

I think that I need to make some key infrastructural aspects are set up first so that people can sort definitions by language.

In the meantime I think if you'd like to start tackling some specific definitions, you could do a PR and build off the English one. Having a few definitions in another language would help us to test our proposed structures.

**Comment by Tagada85** [comment]
Great! I'll take a look at some definitions I might find *easier* to translate. Do you have any tips or advice to make sure I don't change in any way the sense of the definition?

I'm not quite sure how to phrase that question. Are there any resources that you use when constructing a definition in English? Something to help you grasp the actual meaning of words and that could guide me towards the most appropriate translation?

Let me know if that makes any sense 😄

**Comment by sophieschoice** [comment]
Following this discussion, since I talked with Tatiana about translating into Dutch :)

**Comment by tatianamac** [comment]
> Do you have any tips or advice to make sure I don't change in any way the sense of the definition?
For terms already defined in English, I think that it makes the most sense to try to capture the general meaning rather than translating word for word. I recognise as a multi-lingual person that it's more important to capture the spirit of the definition than same definition verbatim.
> I'm not quite sure how to phrase that question. Are there any resources that you use when constructing a definition in English? Something to help you grasp the actual meaning of words and that could guide me towards the most appropriate translation?
Typically I try to read quite a few definitions _and_ articles before starting to write a definition. 

Often, I'll start with a decent core definition and copy and paste it. Then I will edit and adjust it for clarity, nuance, and completeness.

For articles that I think are strong, I will include them in the `Further Reading` section so that people can trace back the original authors. When checking those resources, I prioritise articles by the people who self-identify by the term. For example, I look for articles about polyamory from people who participate in polyamory.

I want to invite more people into the definition-crafting process. With that said, I want to acnowledge that it takes a baseline understanding of how to write sensitively and thoughtfully in what is a very nuanced space. As such, I might suggest starting with translations of existing words first and diving into original French definitions once you've felt more confident.

Hope that answers all the questions.

**Comment by sophieschoice** [comment]
Excellent!

I will make pull request (PR) to get started with Dutch when I have translated a couple of terms :)

**Comment by tatianamac** [comment]
@sophieschoice , you may find @Tagada85 's [PR for French translations](https://github.com/tatianamac/selfdefined/pull/81) helpful? Perhaps you could set it up similarly.

Also, it'd be awesome if you know someone who speaks Dutch who might be able to act as your peer reviewer in my stead?

If not, no worries at all, I can ask on Twitter. 🤗

**Comment by sophieschoice** [comment]
Sounds like a good guidance! I will try to do it similarly, existing github-skills permitting :P 

And I will ask around for a peer reviewer in my network!

**Comment by tatianamac** [comment]
Closing this as the question has been answered.

Translations are accounted for in API spec in #72.
New PR #81 for French.
Future issue to be opened for Dutch.
New issue #82 for documentation on how to write definitions.
